packages/reference/src/components/ResourceEntityErrorCard/ResourceEntityErrorCard.tsx

```diff
@@ -14,11 +15,24 @@ type ResourceEntityErrorCardProps = {
 };
 
 export function ResourceEntityErrorCard(props: ResourceEntityErrorCardProps) {
+  const providerName = getProviderName(props.linkType);
```

Comment:
we could move this after the `isUnsupportedError` check, as it's not used there :)